onnxruntime/core/providers/js/operators/concat.cc

```diff
@@ -6,41 +6,42 @@
 namespace onnxruntime {
 namespace js {
 
-ONNX_OPERATOR_VERSIONED_KERNEL_EX(
-    Concat,
-    kOnnxDomain,
-    1, 3,
-    kJsExecutionProvider,
-    (*KernelDefBuilder::Create())
-        .TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
```

Comment:
Please reference here https://github.com/microsoft/onnxruntime/pull/16901/files#diff-a8e22b7e3708f7ae216482ad10f0bbc8e61ce562448bb299b4cf199d8b557251R15.
It seems like you can directly add the `int32_t` type as below
```
.TypeConstraint("T", DataTypeImpl::GetTensorType<float>()).TypeConstraint("T", DataTypeImpl::GetTensorType<int32_t>())

Reply:
@qjia7 Please don't pick up from the PRs under review. Look at the code base. @fs-eire replied on that PR that it does not work. Data type cannot be both float and in32_t at the same time.

Reply:
@satyajandhyala Thanks for pointing it out. My original meaning maybe like this one https://github.com/microsoft/onnxruntime/pull/16952/files#diff-d8d6d238bdce4bb5db019a9aec28bbedc6dfcce10f5d522268e2f05be83753f5R27. I just thought we can register different types together.